modules/catalog-compaction/src/integrationTest/java/org/apache/ignite/internal/catalog/compaction/ItCatalogCompactionTest.java

```diff
@@ -98,7 +98,8 @@ protected String getNodeBootstrapConfigTemplate() {
                 + "  },\n"
                 + "  clientConnector.port: {},\n"
                 + "  rest.port: {},\n"
-                + "  compute.threadPoolSize: 1\n"
+                + "  compute.threadPoolSize: 1,\n"
+                + "  failureHandler.dumpThreadsOnFailure: false\n"
```

Comment:
I did this based on Ignite 2's experience, but I think it can be changed in the future. The first step is using `stopNode` failure handler by default instead of `noop` [https://issues.apache.org/jira/browse/IGNITE-21456] For now, there are a lot of `green` tests on TeamCity that actually trigger failure handler, and so it will lead to huge log files that will not be analyzed (test are still `green`...)

Reply:
IMHO, we need to change the default value to `stopNode` policy, it makes sense for production deployments.
On the other hand, our tests may use another approach as you suggested. Let's discuss this topic with Yuriy.